src/pyemd/emd.pyx

```diff
@@ -3,7 +3,7 @@
 # distutils: language = c++
 # emd.pyx
 
-from pkg_resources import parse_version
+from packaging.version import parse as parse_version
```

Comment:
Just FYI, there's no need to use an external library, since NumPy itself includes functionality for version comparison - [`NumpyVersion`](https://numpy.org/doc/stable/reference/generated/numpy.lib.NumpyVersion.html).